CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ _Nothing yet..._
 
 ## [4.0.0] - 2020-02-29
 
+- Update logging to note the current transcription number and minor testing config changes
```

Comment:
4.0.0 has already been released. I think you mean to add this in the UNRELEASED section...